dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/validation/EnrollmentImportValidationTest.java

```diff
@@ -91,6 +91,25 @@ void testEnrollmentValidationOkAll()
         assertEquals( TrackerStatus.OK, trackerImportReport.getStatus() );
     }
 
+    @Test
+    void testEnrollmentValidationFailMissingOrgUnitDeleteStrategy()
+        throws IOException
+    {
+
+        TrackerImportParams params = createBundleFromJson( "tracker/validations/enrollments_te_enrollments-data.json" );
+        params.setImportStrategy( TrackerImportStrategy.CREATE );
+        TrackerImportReport trackerImportReport = trackerImportService.importTracker( params );
+        assertEquals( 0, trackerImportReport.getValidationReport().getErrors().size() );
```

Comment:
can you please replace it with `assertFalse(trackerImportReport.getValidationReport().hasErrors())`